qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -55,7 +55,6 @@
                 stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
             usb_type_ehci = usb-ehci
-            bus_tablet1 = usb1.0
```

Comment:
@qingwangrh @YongxueHong Could you help review or forward to the right person if i am wrong ? Thanks!